docs/src/components/table/index.css

```
@@ -0,0 +1,54 @@
+.table {
+  position: relative;
+  width: 100%;
+  margin: 0.5em 0;
+  overflow: auto hidden;
+  border: 0.0625rem solid var(--neutral-05);
+  border-radius: var(--rounded-sm);
+  corner-shape: squircle;
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)